tests/testthat/test-lint.R

```diff
@@ -3,8 +3,6 @@ if (requireNamespace("lintr", quietly = TRUE)) {
   test_that("Package Style", {
     lintr::expect_lint_free(linters = c(lintr::trailing_whitespace_linter,
                                         lintr::no_tab_linter,
-                                        lintr::T_and_F_symbol_linter,
```

Comment:
These should be kept in

Reply:
They are available if you install the development version of the lintr package https://github.com/jimhester/lintr